src/UI/state/configSlice.ts

```diff
@@ -19,14 +19,20 @@ export interface ConfigState {
       zIndex: number
     }[]
   }
+
+  map_overlays: {
```

Comment:
This isn't a config thing really - I think we want a new slice for map-specific bits of the store. Can you move this and the styles to a new slice?